### ISSUE TITLE
I've update the bundle and I am getting the error below(Argument #2 ($area) must be of type string)

### ISSUE THREAD

**Issue by LeikoDmitry** [nelmio/NelmioApiDocBundle#2193]
`Nelmio\ApiDocBundle\Render\RenderOpenApi::render(): Argument #2 ($area) must be of type string, null given, called in /app/vendor/nelmio/api-doc-bundle/Render/RenderOpenApi.php on line 61`

My config file

```
nelmio_api_doc:
    documentation:
        servers:
            - url: https://localhost
              description: This is just default server
        info:
            title: Publisher API
            description: This is an awesome app!
            version: 1.0.0
        components:
            securitySchemes:
                Bearer:
                    type: http
                    scheme: bearer
                    bearerFormat: JWT
        security:
            - Bearer: []
                
    areas:
        path_patterns:
            - ^/api/v1
```

**Comment by lucasmirloup** [comment]
Hi @LeikoDmitry
I've had the same problem on a project of mine.
It was caused by our code that set custom attributes on the Symfony Request.

We used a `onKernelRequest` listener that contained this code :
```php
$this->requestStack->getCurrentRequest()?->attributes->set('area', $thisVariableCanBeNull);
```

At the time, I wasn't aware that custom Request attributes can (and will) be used by Symfony to set controller parameters.

Because [`OpenApiRender::render` controller route](https://github.com/nelmio/NelmioApiDocBundle/blob/2f20162e03f0aae333eebf569c9a231da1f327d3/src/Render/RenderOpenApi.php#L76) has a controller parameter named `$area`, our Request attribute named `'area'` overrode the default value of this parameter.

Imo, that GitHub issue can be closed :ok_hand:

Cc @DjordyKoert
PS: Thank you for maintaining this nice library :+1:
